tools/talis/download.go

```diff
@@ -11,6 +12,14 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// Regex to match ANSI escape codes
+var ansiEscape = regexp.MustCompile(`\x1b\[[0-9;]*[a-zA-Z]`)
+
+// stripANSI removes ANSI escape codes from the input string, returning a plain text version without formatting codes.
+func stripANSI(input string) string {
+	return ansiEscape.ReplaceAllString(input, "")
+}
```

Comment:
definitely not blocking, but typically I like to put utils at the bottom of the file and the important stuff at the top